mkdocs.yml

```diff
@@ -43,7 +43,8 @@ nav:
             - 'Azure_IoT_Central_integration/Device_operations/Exporting_devices_to_Azure_IoT_Central.md'
     - 'Concepts':
         - 'Concepts/LwM2M_mappings.md'
-
+    - 'Use cases':
```

Comment:
I think that this section should be named "Tutorials", "Examples", "How-to guides" all of them are better than use cases, because the use case is a very business phase. I expect different verticals descriptions or customer success stories when I think about use cases, but this is a tutorial.

Reply:
ok